barretenberg/bbup/README.md

```diff
@@ -0,0 +1,54 @@
+# BBup
+
+BBup is a CLI tool that makes it easy to install the [Barretenberg](https://github.com/AztecProtocol/aztec-packages/blob/master/barretenberg/README.md) proving backend.
+
+It assumes you are using [Noir](https://noir-lang.org) as the frontend language.
```

Comment:
```suggestion
```

Reply:
It does assume we are using Noir, as per the behaviour discussed below